lib/vk/schema/types/groups_role_options.rb

```diff
@@ -0,0 +1,10 @@
+require 'vk/schema/types'
```

Comment:
WTF, thoughtbot? Why should I prefer double quoted strings if I don’t need to interpolate anything?